storage/innobase/xtrabackup/test/bootstrap.sh

```diff
@@ -143,11 +145,19 @@ main () {
             tarball="Percona-Server-${VERSION_PREFIX}-rel${VERSION_SUFFIX}-Linux.${arch}.ssl$(ssl_version).tar.gz"
             ;;
         xtradb57)
+            short_version="$(echo ${VERSION} | awk -F "." '{ print $3 }' | cut -d '-' -f1)"
             url="https://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-${VERSION}/binary/tarball/"
-            if [[ $(echo ${VERSION} | awk -F "." '{ print $3 }' | cut -d '-' -f1) -lt "31" ]]; then
+            if [[ ${PXB_TYPE} == "Debug" ]] || [[ ${PXB_TYPE} == "debug" ]]; then
```

Comment:
why not just use `tr '[:upper:]' '[:lower:]'` for the variable and after that only compare with `debug`?